### PR TITLE
Change warnings to come from DB

### PR DESCRIPTION
**Pull request by farridav**

### CODE REVIEW

File: django_warnings/models.py [hunk 1]
```diff
@@ -20,6 +23,14 @@ class Warning(models.Model):
     content_type = models.ForeignKey(ContentType)
     object_id = models.PositiveIntegerField()
     content_object = GenericForeignKey('content_type', 'object_id')
+    identifier = models.CharField(
+        max_length=50, choices=getattr(settings, 'warning_codes', []),
+        blank=True, null=True
+    )
```
Comment (JoseKilo) [inline django_warnings/models.py]: I would prefer it to be `TextField`. At least, for consistency: every other text-like field in this model is TextField

File: django_warnings/models.py [hunk 2]
```diff
@@ -32,6 +43,12 @@ class Warning(models.Model):
     last_acknowledger = models.PositiveIntegerField(null=True, default=None)
     last_acknowledged = models.DateTimeField(null=True, default=None)
 
+    class Meta:
+        ordering = ('last_generated',)
```
Comment (JoseKilo) [inline django_warnings/models.py]: We should document this default ordering in the README.

Normally I'm against default orderings, but this one sounds like a legitimate case

File: django_warnings/models.py (hunk 1 above)
Comment (farridav) [inline django_warnings/models.py]: sure, +1 for consistency :)

**Comment by JoseKilo** [comment]
LGTM :+1:

**Pull request merged**